### DISCUSSION TITLE
Multi-dimensional time series data with MSTUMP for mixture of categorical and continues data

### DISCUSSION THREAD

**Discussion by Sandy4321** [TDAmeritrade/stumpy#60] [Q&A]
can be Multi-dimensional time series with mixture of categorical and continues data be predicted
by 
Multi-dimensional time series data with MSTUMP:

for example at each second we have vector of observations:
t1 [ red, 0.6 , big, 123]
t2[green, 1.7 , small , 3345]
etc?

**Comment by seanlaw** [comment]
No, STUMPY can only be used with floating point numbers. To be absolutely clear, this is not a limitation of STUMPY. It is simply not possible to compute a z-normalized Euclidean distance (which relies on computing the mean and standard deviation) for categorical values. I hope that helps clarify things

**Comment by seanlaw** [comment]
@Sandy4321 It looks like you've asked this same question in the past in #38 (and, hopefully, the answer was conclusive). Closing this now.
